test/integration/iam-policy-management.v1.test.js

```diff
@@ -40,7 +40,7 @@ describe('IamPolicyManagementV1_integration', () => {
   let testV2PolicyId;
   let testAssignmentPolicyId;
   let testTargetAccountId;
-  let testTargetEnterpriseAccountId;
```

Comment:
missing test for listActionControlTemplateAssignments()

Reply:
I guess we are referring this `listActionControlAssignments` and we have a testcase for the same

Reply:
ah, I was thrown off by the different names of the functions:
* createActionControlTemplateAssignment   <<< contains "Template"
* getActionControlAssignment
* updateActionControlAssignment
* deleteActionControlAssignment
* listActionControlAssignments